src/main/java/com/faforever/client/map/MapService.java

```diff
@@ -266,6 +224,52 @@ private Thread startDirectoryWatcher(Path mapsDirectory) {
     return thread;
   }
 
+  private boolean checkScenarioLua(Path mapPath) {
```

Comment:
Normally we don't make comments but as this is not a common thing to do maybe explain it in a java doc

Reply:
are there examples of how you think it should be documented

Reply:
Added in the javadoc